extension/js/content_scripts/webmail/setup-webmail-content-script.ts

```diff
@@ -280,11 +282,30 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
     }
   };
 
+  const updateClientConfiguration = async (acctEmail: string) => {
+    try {
+      const authInfo = await AcctStore.authInfo(acctEmail);
+      await (new AccountServer(acctEmail)).accountGetAndUpdateLocalStore(authInfo);
+    } catch (e) {
+      if (e instanceof BackendAuthErr) {
+        // user will see a prompt to log in during some other actions that involve backend, and therefore authInfo.uuid will eventually be filled,
+        // at which point the update will happen next time user loads the page
+      } else if (ApiErr.isNetErr(e)) {
+        // ignore
+      } else {
+        Catch.reportErr(e);
+        Ui.toast(`Failed to update FlowCrypt Client Configuration: ${e instanceof Error ? e.message : String(e)}`);
+      }
+    }
+
+  };
+
   const entrypoint = async () => {
     try {
       const acctEmail = await waitForAcctEmail();
       const { tabId, notifications, factory, inject } = await initInternalVars(acctEmail);
       await showNotificationsAndWaitTilAcctSetUp(acctEmail, notifications);
+      await updateClientConfiguration(acctEmail);
```

Comment:
@tomholub I'm afraid, in some cases (on a network error) it may considerably delay the gmail replacer so that it'll be annoying to the user?

Reply:
Yes, I'd like it to happen async, like EKM refetch.

Reply:
Sure. I'm ok with that. I don't necessary need right-now immediate change, what I need is that changes do eventually get propagated.

Reply:
Done